osbrain/proxy.py

```diff
@@ -100,6 +100,30 @@ def _ready_or_timeout(self, time0, timeout):
             raise
         return True
 
+    def wait_for_running(self, timeout=0):
```

Comment:
Also, I think a default value that is not 0., but greater, makes more sense. Waiting really forever is rarely useful.

Maybe `3.`? I think we currently use that timeout by default in other operations.

Reply:
I thought the timeout should be optional,  but now that you mention it, if the agent  is not ready after 3 seconds, you probably have some other problem, and the agent is likely to never be ready at all. I'll change it to 3, then :smile: